README.md

```diff
@@ -1,12 +1,12 @@
 # usi-arrayexpress
 
-This repository contains modules for transformation of experimental metadata between [MAGE-TAB](fged.org/projects/mage-tab/) and JSON format used in the [Unified Submissions Interface (USI)](https://github.com/EMBL-EBI-SUBS). 
+This repository contains modules for transformation of experimental metadata between [MAGE-TAB](http://fged.org/projects/mage-tab/) and JSON format used in the [Unified Submissions Interface (USI)](https://github.com/EMBL-EBI-SUBS). 
 
 
 ## Installation and requirements
 
 Python 3.6 interpreter<br>
-Package requirements:git 
+Package requirements:
```

Comment:
might be good to have this on a requirements.txt file and make the whole thing pip installable in the near future.

Reply:
Yes, there is a requirements.txt too: https://github.com/ebi-gene-expression-group/usi-arrayexpress/blob/develop/requirements.txt We've made that for the Travis integration. I am keeping the package creation still in the back of my mind :)